net/lp_rpc.proto

```diff
@@ -20,12 +20,62 @@ message TranscoderRequest {
   bytes sig   = 2;
 }
 
+/*
+  OSInfo needed to negotiate storages that will be used.
+  It carries info needed to write to the storage.
+*/
+message OSInfo {
+
+  // Storage type: s3, ipfs.
+  string storage = 1;
+
+  S3OSInfo s3info = 16;
+}
+
+message S3OSInfo {
+
+  // Configured S3 bucket name
+  string bucket = 1;
+
+  // Configured S3 region
+  string region = 2;
```

Comment:
Can we collapse both fields into a single host? Eg, `http://bucket.s3-aws-region.amazonaws.com`

Reply:
probably, need to check. but why would we want to?

Reply:
It's fewer fields, and we aren't making assumptions about the URL structure on the remote side. We only need to create the URL once at the owner, and have the remote use that. Is there a reason for the remote to be explicitly aware of the bucket and/or region?